doc/community-release-build.md

```diff
@@ -0,0 +1,202 @@
+  * Upgrade from previous release
+* Tagging - Apply community release Tag (like v0.11.0) to the latest CommitId on the community release development branch.
+* Prepare Community release - create PR in **community-operators-prod** repo (based on manifests from 3scale-operator repo), 
+as in [PR example](https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/4150)
+  * **Merge of this PR will publish Release**.
+* Do sanity test of the Release after publishing
+
+## Versioning Strategy
+Semantic Versioning scheme will be used for Community releases:  MAJOR.MINOR.PATCH.
+* MAJOR version: Increments when breaking changes are introduced
+* MINOR version: Increments when backward-compatible functionality is added.
+* PATCH version: Increments when backward-compatible bug fixes are made.
+* Community releases Versions examples:  0.8.2,  0.9.0,  0.10.1,  0.11.0
+* The Version is used in Community releases Tag name
+
+```shell
+$ git tag |grep -E "0.9.0|0.10.1|0.11.0"
+v0.10.1
+v0.11.0
+v0.9.0
+```
+
+* Relation to Product release versions
+  * It will be  defined in community-operators-prod/operators/3scale-community-operator, as bundle channel 
+  in `annotations.yaml` and `budle.Dockerfile`; for example:
+```
+0.10.1/bundle.Dockerfile:LABEL operators.operatorframework.io.bundle.channels.v1=threescale-2.13
+0.10.1/metadata/annotations.yaml:  operators.operatorframework.io.bundle.channels.v1: threescale-2.13
+```
+
+* It’s recommended also to add the annotation to release Tag, as in example:
+```  
+$ git tag -n v0.11.0
+v0.11.0         Community Release branch 3scale-community-v0.11.0, based on product 3scale-2.14.1-GA
+```
+
+## Steps to Perform Community Release in 3scale-operator repo
+1. Community Release Development Branch
+
+* Create a Community release development branch **based on Product Release CommitID**
+* Create PR and target it to **3scale-2.X-stable** branch
+
+```
+$ cd 3scale-operator
+$ git checkout -b 3scale-community-v0.11.0 bf398d34ae9378befc4e6e8bf447adbeef37c054
+```
+2. 3scale Components Images
+* Community release is using its own Component images,for:
+    - apisonator
+    - apicast
+    - porta
+    - zync
+
+3. Create PR in 3scale-operator upstream for new bundle
+   * Work to have E2E tests passed in PR
+   * Merge PR to **3scale-2.X-stable** branch
+  
+4. Testing
+   
+   * Testing of `3scale-operator` must be completed before open PR in `community-operators-prod` repo
+   * Testing must be completed before Tagging
+   * Testing must be done for `Fresh Install` and `Upgrade`
+  
+
+5. Tagging
+
+- Community Release is defined by Tag.  
+- Tag will be applied to the Community Release Branch
+- Tag will be applied after Testing completion.
+
+```
+$ cd 3scale-operator
+$ git tag -a v0.11.0  -m "Community Release branch 3scale-community-v0.11.0, based on product 3scale-2.14.1-GA" ab1783b4207e43480bf7538d62ecbd83636ecf0c
+$ git push myfork v0.11.0
+```
+
+* Compare Product and Community release tags
```

Comment:
@valerymo Is it worth adding something here to explain why we are doing the comparison and what actions might be taken based on the results of the comparison

Reply:
Thank you for comment. I looked again now, and seems to me it's not needed; removed.